spec/ParseGraphQLServer.spec.js

```diff
@@ -9450,7 +9452,8 @@ describe('ParseGraphQLServer', () => {
               body: body2,
             });
             expect(res.status).toEqual(200);
-            const result2 = JSON.parse(await res.text());
+            const resText = await res.text();
```

Comment:
```suggestion
            const result2= await res.json();
```
`res.json` already parses it as JSON.

Reply:
yes good catch useless JSON.parse